src/main/java/com/example/gamelist/GameController.java

```diff
@@ -25,12 +26,19 @@ public  List<Game> gamelist(){
 
 
     @PostMapping("/gamelist")
-    public ResponseEntity<Response> create(@RequestBody GameForm form){
+    public ResponseEntity<GamelistService.Response> create(@RequestBody GameForm form){
         URI url = UriComponentsBuilder.fromUriString("http://localhost:8080")
                 .path("/gamelist/id")
                 .build()
                 .toUri();
-        return ResponseEntity.created(url).body(new Response("name successfully created"));
+        return ResponseEntity.created(url).body(new GamelistService.Response("name successfully created"));
+
+    }
+    @PatchMapping("/gamelist/{id}")
```

Comment:
メソッドとメソッドの間には空行入れましょう。
```suggestion
    }
    
    @PatchMapping("/gamelist/{id}")
```

Reply:
<img width="901" alt="スクリーンショット 2023-12-16 23 41 46" src="https://github.com/otyacazyu/gamelist/assets/143002834/7c1203ac-d7db-4dcc-b1b8-79288c39257a">

空行を削除を詰めました

Reply:
逆です。
空行を入れてほしいです。

Reply:
[045311d](https://github.com/otyacazyu/gamelist/pull/7/commits/045311dbdcf6e64681490c71bdb50ad16f570b5e)

失礼しました。
空行を追加を行いました。

Reply:
1 コミットに修正内容が多すぎてわかりづらいですね。
https://github.com/otyacazyu/gamelist/pull/7#discussion_r1429013444
こちらの内容を確認して対応お願いします。

空行追加についてはOKです。